test/master_o365collector_test.js

```diff
@@ -198,295 +196,4 @@ describe('Master Function o365collector.js Units', function() {
         });
     });
 
-    describe('O365 collector checkin tests', function() {
```

Comment:
Could you please double-check that all these checkin cases are covered by `_checkEnableAuditStreams` tests?